docs/document/docs/guides/format-setting-plugin.md

```diff
@@ -57,6 +57,37 @@ src/
 ├── manifest.json
 ```
 
+## Create a Plug-in Icon
+- The icon image for the Plug-in.
+- This file is required.
+- Use the following file types: png, jpg, gif, bmp
+- The maximum file size is 20MB.
+- Please add icon file into folder ``image/icon.png``
+
+## Create a config.css file
+- The stylesheet initiated on the Plug-in.
+- You can follow the Kintone stylesheet if you want to use the Kintone design.
+- The maximum file size is 20MB.
+- This is optional, and can be ignored. But following the example below, You can create a file ``css/config.css``
+
+```css
+.kuc-dropdown__group > .kuc-dropdown__group__toggle {
+  width: 200px;
+}
```

Comment:
I think this won't work now because the class name will include the version number now.
I think we can remove this setting.